app/Services/ReportbackService.php

```diff
@@ -41,7 +41,7 @@ public function create($data, $transactionId)
 
         // POST reportback back to Phoenix, unless told not to.
         // If request fails, record in failed_jobs table.
-        if (! $data['do_not_forward']) {
+        if (! isset($data['do_not_forward'])) {
```

Comment:
won't `! isset($data['do_not_forward'])` always evaluate to `false` if `$data['do_not_forward']` is set as `true` or `false`?